src/components/ui/form.tsx

```diff
@@ -106,6 +108,117 @@ export function FormField({
   );
 }
 
+type SelectFieldProps = SelectHTMLAttributes<HTMLSelectElement> & {
```

Comment:
high level question: do we really want all the form elements in a single file? Wouldn't one file per element be cleaner?

Reply:
In practice I don't think it adds that much value to spread stuff over different files and then re-exporting them through an index file (modern editors and TypeScript make it trivial to navigate). But I'm open to change if you feel strongly about it.